src/vm/eventpipe.h

```diff
@@ -253,16 +255,11 @@ class EventPipe
     static EventPipeSessionID Enable(
         LPCWSTR strOutputPath,
         uint32_t circularBufferSizeInMB,
-        uint64_t profilerSamplingRateInNanoseconds,
+        uint32_t profilerSamplingRateInNanoseconds,
```

Comment:
Is there a particular reason we're changing this to uint32_t?

Reply:
This is a parameter to the `SampleProfiler::SetSamplingRate(unsigned long)` method. The parameter is 32-bits unsigned integer. Should I make the it 64-bits?

Reply:
I'm not sure... If we set this to 32 bits then the absolute minimum sampling rate we can set is once per 2^32 nanoseconds which translates to ~4 seconds. Is that ridiculously low enough to set as the absolute minimum?

Reply:
I'm open with either 32-bits or 64-bits. I updated it because it is used on the `SampleProfiler`, and it gets truncated anyways.

Reply:
/cc @noahfalk